ironfish/src/mining/poolMiner.ts

```diff
@@ -51,7 +52,7 @@ export class MiningPoolMiner {
       port: options.port,
       logger: options.logger,
     })
-    this.stratum.onConnected.on(() => this.stratum.subscribe(this.publicAddress))
+    this.stratum.onConnected.on(() => this.stratum.subscribe(this.publicAddress, options.name))
```

Comment:
nit: since we save the `publicAddress` on the miner can we save the name on here too for consistency? So you can use `miner.name` as well?